src/main/webapp/scriptler.css

```diff
@@ -13,6 +13,7 @@
 .scriptler-link-button {
   background: none;
   border: none;
+  cursor: pointer;
```

Comment:
Concerns the catalog save icon
![screenshot_2018-01-11_224301_001](https://user-images.githubusercontent.com/2662497/34848924-eda3cdc2-f720-11e7-894d-35fd6e596993.png)